views/popups.pug

```diff
@@ -65,6 +65,21 @@
         button.btn.btn-link.no(type="button" data-dismiss="modal") No
         button.btn.btn-link.yes(type="button" data-dismiss="modal") Yes
 
+#enter-something.modal.fade.gl_keep(tabindex="-1" role="dialog")
```

Comment:
Is this a new thing? From the discussion with @partouf is this you picking up Partouf's separate change and rolling it in? It's :100: ok I'm just trying to understand.

Reply:
Yeah, to reduce the amount of merges @partouf has to suffer through with #2725 we decided to move in his alert.js changes here

Reply:
Yes, I made changes in alert.js for the multifileui to rename files, which includes a new modal dialog. @supergrecko was nice enough to implement them here so I won't be scratching my head on how to merge a deleted js files contents etc.